searches/linear_search.py

```diff
@@ -66,8 +68,8 @@ def rec_linear_search(sequence: list, low: int, high: int, target: int) -> int:
 
 
 if __name__ == "__main__":
-    user_input = input("Enter numbers separated by comma:\n").strip()
-    sequence = [int(item.strip()) for item in user_input.split(",")]
+    user_input = input("Enter numbers separated by space\n")
```

Comment:
```suggestion
    user_input = input("Enter numbers separated by space\n").strip()
```
Same here, why get rid of the `.strip()`?